manuscript/draft.Rmd

```diff
@@ -166,3 +177,83 @@ None of the models are true forcasts as they use observed predictor variables (e
 * Need predictions at multiple time scales
 * Prediction quality at short time scales might not be a good indicator of 
   long-term effectiveness
+
+
+
+## Box 1: Ten simple rules for making and evaluating ecological forecasts
+
+### 1. Compare multiple modeling approaches
+
+Typically ecological forecasts use one modeling approach or a small number of
+related approaches. By fitting and evaluating multiple modeling approaches we
+can learn more rapidly about the best approaches for making predictions for a
+given ecological quantity. This includes comparing process based and data-driven
+models.
+
+### 2. Use time-series data when possible
+
+Forecasts describe how systems are expected to change through time. While some
+areas of ecological forecasting focus primarily on time-series data, others
+primarily focus on using spatial models and space-for time substitutions. Using
+ecological and environmental time-series data allows the consideration of actual
+dynamics from both a process [@treddenick2016a] and error structure perspective.
+
+### 3. Pay attention to uncertainty
+
+Understanding confidence in a forecast is just as important as understanding the
+average or expected outcome. Failing to account for uncertainty can result in
+overconfidence in highly uncertain outcomes leading to poor decision making and
+erosion of confidence in ecological forecasts. Models should explicitly include
+sources of uncertainty where possible and evaluations of forecasts should assess
+the accuracy of uncertainties as well as point estimates [@dietz2017].
+
+### 4. Use predictors related to the question
+
+Many ecological forecasts use data that is readily available and easy to work
+with. While ease of use is a reasonable consideration it is also important to
+include predictor variables that are expected to relate to the ecological
+quantity being forecast and dynamic time-series of predictors instead of
+long-term averages. Investing time in identifying and acquiring better predictor
+variables may have at least as many benefits as using more sophisticated
+modeling techniques.
+
+### 5. Assess how forecast accuracy changes with time-lag
+
+In general the accuracy of forecasts decreases with the length of time into the
+future being forecast [@petch2015]. This decay in accuracy and associated
+forecast horizons should be considered when evaluating forecasts and comparing
+models.
```

Comment:
Also important because the most accurate model may not be the same at every time step. Ward et al 2014 has some subtle examples of this in their figures.